dev-python/pyxml/pyxml-0.8.4.recipe

```diff
@@ -1,20 +1,94 @@
-DESCRIPTION="Python XML package"
+SUMMARY="Python2 XML package"
+DESCRIPTION="Python XML comes with a validating XML parser, an implementation of the SAX \
+and DOM programming interfaces, an interface to the Expat parser (and the Expat parser \
+itself), and a C helper module that can speed up xmllib.py by a factor of 5."
 HOMEPAGE="http://pyxml.sourceforge.net/topics/index.html"
-SOURCE_URI="http://sourceforge.net/projects/pyxml/files/latest/download?source=files"
-CHECKSUM_MD5="1f7655050cebbb664db976405fdba209"
+COPYRIGHT="2000 Fourthought Inc."
+LICENSE="Python"
 REVISION="1"
-STATUS_HAIKU="untested"
-DEPEND="dev-lang/python >= 2.6.4"
+SOURCE_URI="https://downloads.sourceforge.net/project/pyxml/pyxml/$portVersion/PyXML-$portVersion.tar.gz"
+CHECKSUM_SHA256="9fab66f9584fb8e67aebd8745a5c97bf1c5a2e2e461adf68862bcec64e448c13"
+SOURCE_DIR="PyXML-$portVersion"
+
+ARCHITECTURES="any"
```

Comment:
Since `gcc` is used, this package contains architecture-specific bits, please expand this to multiple architectures.